tests/src/dir.targets

```diff
@@ -120,7 +119,8 @@
 
   <Target Name="CopyNativeProjectBinaries">
      <ItemGroup>
-        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" />
+        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" Condition="'$(BuildOS)' == 'Windows_NT'"/>
+        <NativeProjectBinaries Include="$(ProjectDir)\..\bin\Product\$(OSPlatformConfig)\bin\*.*" Condition="'$(BuildOS)' != 'Windows_NT'"/>
```

Comment:
see how --testNativeBinDir is being used in https://github.com/dotnet/coreclr/blob/5f05a462321923ee6d9975cc3f88a233b3143332/tests/runtest.sh#L18